extension/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "extension",
+  "version": "1.0.0",
```

Comment:
```suggestion
  "version": "alpha",
```

Reply:
https://semver.org/spec/v1.0.0-beta.html should include a version too
Should i switch to 1.0.0-alpha